### PR TITLE
OCPBUGS-31706: ubxtool cold boot has no effect on logs and metrics

### PR DESCRIPTION
**Pull request by aneeshkp**

### CODE REVIEW

File: pkg/daemon/gpsd.go [hunk 1]
```diff
@@ -252,21 +257,18 @@ retry:
...
-				if g.Stopped() {
-					doneFn()
-					return
-				}
 				g.offset = nOffset
-				if nStatus < 3 {
-					g.sourceLost = true
-				} else if nStatus >= 3 && g.isOffsetInRange() {
+				g.sourceLost = false
+				switch nStatus >= 3 {
```
Comment (jzding) [inline pkg/daemon/gpsd.go:262]: It is simpler to use if-else

Comment (aneeshkp) [reply pkg/daemon/gpsd.go:262]: This is better for readability

File: pkg/event/stats.go [hunk 2]
```diff
@@ -100,6 +104,32 @@ func (d *Data) AddEvent(event EventChannel) {
...
+}
+
+// toString ... data details
+func (dd DDetails) toString() string {
+	out := strings.Builder{}
+	for _, d := range dd {
+		out.WriteString("  Iface name: " + d.IFace)
+		out.WriteString("  state: " + string(d.State))
+		out.WriteString("  clock type: " + string(d.ClockType))
+		out.WriteString("  process name: " + strconv.FormatBool(d.sourceLost))
```
Comment (jzding) [inline pkg/event/stats.go:130]: out.WriteString(" signal source: " + string(d.signalSource))
out.WriteString(" source lost: " + strconv.FormatBool(d.sourceLost))

Comment (aneeshkp) [reply pkg/event/stats.go:130]: Done

Comment (jzding) [reply pkg/event/stats.go:131]: I don't see it

Comment (aneeshkp) [reply pkg/event/stats.go:131]: click on  view view changes

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**